recipe/meta.yaml

```diff
@@ -54,7 +56,10 @@ test:
     - typing-extensions >=3.7.4  # [py<310]
   commands:
     - pip check
-    - pytest tests -vv -k "not (test_contents_unstaged or test_wheel.py or test_pep518 or test_tag or test_detect_wheel_tag or test_spam)" # tests are relevant to wheel and not conda packages
+    # CPPFLAGS exports -DNDEBUG which is unwanted
+    - export CPPFLAGS=${CPPFLAGS//-DNDEBUG/}  # [unix]
+    # tests are relevant to wheel and not conda packages
+    - pytest tests -vv -k "not (test_editable_install or test_tags.py or test_wheel.py or test_mesonpy_meta_finder or test_editble_reentrant or test_spam)" # tests are relevant to wheel and not conda packages
```

Comment:
SHouldn't `test_editable_install`, `test_mesonpy_meta_finder`, `test_editble_reentrant` and `test_spam` work with meson 1.2?

Reply:
https://cloud.prefect.io/anaconda/flow-run/8be1e37c-022d-4eed-9fad-508baa19ad0c?logs=&logId=bfb0208d-4156-46c1-a795-a937a95aefa6